sdk/attestation/azure-security-attestation/README.md

```diff
@@ -1,39 +1,362 @@
+Attestation is a process for demonstrating that software binaries were properly instantiated on a trusted platform. Remote 
+relying parties can then gain confidence that only such intended software is running on trusted hardware. 
 
-## Getting started
+Azure Attestation enables cutting-edge security paradigms such as Azure Confidential computing and Intelligent Edge 
+protection. Customers have been requesting the ability to independently verify the location of a machine, the posture 
+of a virtual machine (VM) on that machine, and the environment within which enclaves are running on that VM. Azure 
+Attestation will empower these and many additional customer requests.
 
-For a rich example of a well formatted readme, please check [here.](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md) In addition, this is an [example readme](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-EXAMPLE.md) that should be emulated. Note that the top-level sections in this template align with that of the [template.](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md)
+Azure Attestation receives evidence from compute entities, turns them into a set of claims, validates them against 
+configurable policies, and produces cryptographic proofs for claims-based applications (for example, relying parties 
+and auditing authorities).
+
+# Getting started
+For the best development experience, we recommend that developers use the [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies.
+
+
+## Prerequisites
+
+- [Azure Subscription][azure_subscription]. Sign up for a [free trial](https://azure.microsoft.com/pricing/free-trial/) or use your [MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+- An existing [Azure Attestation instance][azure_attestation]. If you need to create an attestation instance, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-instance-name>` with your own, unique names:
+
+```bash
+az attestation create --resource-group <your-resource-group-name> --name <your-service instance name>
+```
+
+## Download & Install
+
+### Install Dependencies
+
+#### Windows
+
+On Windows, dependencies are managed by [vcpkg](https://github.com/microsoft/vcpkg). You can reference the [Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) to quickly set yourself up.
+After Vcpkg is initialized and bootstrapped, you can install the dependencies:
+```BatchFile
+vcpkg.exe install curl:x64-windows-static
+vcpkg.exe install openssl:x64-windows-static
+```
```

Comment:
The end user shouldn't need to install these dependencies independently.

The acquisition for any C++ client library and Azure SDK package should be:
`vcpkg install <package_name>:<optional triplet>`

And that should bring in the dependencies needed.

Reply:
This text was taken from keyvault's readme. If that is the case, we need to update KV (and likely other readme.md files).